draw.io-api/pom.xml

```diff
@@ -76,6 +79,16 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.63</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.63</version>
+    </dependency>
+    <dependency>
+      <groupId>com.mxgraph</groupId>
+      <artifactId>gae-stub</artifactId>
+      <version>1.0-2</version>
```

Comment:
Do we still need this if we exclude the Java files below? Would be good to add a comment at least.